api/ceramic_cache/models.py

```diff
@@ -0,0 +1,15 @@
+"""Ceramic Cache Models"""
+
+from account.models import EthAddressField
+from django.db import models
+
+
+class CeramicCache(models.Model):
+    address = EthAddressField(null=True, blank=False, max_length=100)
+    provider = models.CharField(
+        null=False, blank=False, default="", max_length=256, db_index=True
+    )
+    stamp = models.JSONField(default=dict)
```

Comment:
create mark as deleted column boolean, once its recreated mark as deleted should be false